frontend/apps/crates/components/src/profile_image/styles.css

```diff
@@ -0,0 +1,73 @@
+.edit-button {
+    grid-row: 1;
+    grid-column: 1;
+    position: relative;
+    left: 60px;
```

Comment:
Instead of using positioning with a hardcoded 60 pixels, you should probably use the following:
```css
justify-self: end;
align-self: start;
```

Also, make sure that the button is at least 32 pixels in size. (needed for touch, fingers are larger than cursors)

```css
min-width: 30px;
min-height: 30px;
```